docs/source/en/chat_templating.md

```diff
@@ -370,7 +371,7 @@ messages = [
 Now, let's apply the chat template and generate a response:
 
 ```python
-inputs = tokenizer.apply_chat_template(messages, chat_template="tool_use", tools=tools, add_generation_prompt=True, return_dict=True, return_tensors="pt")
+inputs = tokenizer.apply_chat_template(messages, tools=tools, add_generation_prompt=True, return_dict=True, return_tensors="pt")
```

Comment:
Does this change because of the removal of "revision" above?

Reply:
No, it's actually been unnecessary for a while! Both it and the "revision" bit were outdated (it still worked with them, but it's much cleaner without)